src/compositions/useTaskRunsHistory.ts

```diff
@@ -0,0 +1,45 @@
+import { SubscriptionOptions, UseSubscription, useSubscriptionWithDependencies } from '@prefecthq/vue-compositions'
+import merge from 'lodash.merge'
+import { computed, ComputedRef, MaybeRefOrGetter, toRef, toValue } from 'vue'
+import { TaskRunHistory } from '..'
```

Comment:
```suggestion
import { TaskRunHistory } from '@/models'
```
I think this is the correct import but I'm not totally sure